crossbeam-channel/src/channel.rs

```diff
@@ -1506,3 +1507,63 @@ pub(crate) unsafe fn read<T>(r: &Receiver<T>, token: &mut Token) -> Result<T, ()
         ReceiverFlavor::Never(chan) => chan.read(token),
     }
 }
+
+/// Channel flavors that are reconnectable, ie, may survive
+/// without senders.
+///
+pub mod reconnectable {
+    // note: we need separate types because the At/Tick/Never
+    // channel types do not support this, but they all use the
+    // same exact Receiver type as array and list flavours.
+
+    // The intended usage of this module is just to replace any `use crossbeam_channel::...`
+    // with `use crossbeam_channel::reconnectable::...` and have everything work out of the box.
+
+    use super::*;
+    pub use crate::*;
```

Comment:
Please expand this glob re-export.

Also, re-exporting `bounded` here will make the future reconnectable bounded channel addition a breaking change.

Reply:
Thanks for spotting this!